src/cardano-wallet-adapter/adapter.ts

```diff
@@ -5,6 +5,7 @@ import { CROSSMINT_LOGO_21x21, CrossmintWalletName } from "../consts/branding";
 import { BlockchainTypes, CrossmintEmbedConfig, CrossmintEmbedParams, CrossmintEnvironment } from "../types";
 import { buildConfig } from "../utils/config";
 import { CardanoWalletApi } from "./types";
+import { WalletPublicKeyError, WalletWindowClosedError } from "@solana/wallet-adapter-base";
```

Comment:
hm, we're using Error classes from `@solana/wallet-adapter-base"` in this Cardano-based adapter which feels weird.
Similarly, not urgent, but would think we should just keep the `Error` we had before